include/maths/CLinearAlgebraEigen.h

```diff
@@ -271,8 +275,58 @@ class CDenseVector : public Eigen::Matrix<SCALAR, Eigen::Dynamic, 1> {
         }
         return seed;
     }
+
+    std::vector<SCALAR> toStdVector() const {
+        std::vector<SCALAR> result;
+        for (int i = 0; i < this->size(); ++i) {
+            result.push_back(this->operator()(i));
+        }
+        return result;
+    }
+
+    static CDenseVector<SCALAR> fromStdVector(const std::vector<SCALAR>& vector) {
+        CDenseVector<SCALAR> result(vector.size());
+        for (auto element : vector) {
+            result << element;
+        }
+        return result;
+    }
+
+    //! Persist by passing information to \p inserter.
+    static void persist(const CDenseVector<SCALAR>& vector,
+                        core::CStatePersistInserter& inserter) {
+        std::vector<SCALAR> temporaryVector;
+        for (int i = 0; i < vector.size(); ++i) {
+            temporaryVector.push_back(vector(i));
+        }
+        inserter.insertValue(DENSE_VECTOR_TAG, core::CPersistUtils::toString(temporaryVector));
```

Comment:
This should be using toStdVector no?

Reply:
The method was replaced by `acceptPersistInserter` which uses `toStdVector` internally.

Reply:
The method was replaced by `acceptPersistInserter` which uses `toStdVector` internally.